tests/unit/drivers/test_flow_visualization.py

```diff
@@ -0,0 +1,9 @@
+from jina.flow import Flow
+
+
+flow = (Flow().add(name='pod_a')
```

Comment:
Different points:

- Pytest only runs functions that start with the name test. so we need a `def test_flow_visualization()`

- This test does not go under `drivers`, it must go under `flow` subfolder in unit tests.

- No need to add ConcatEmbedDriver, it does not affect the flow visualization.

- We need some assertion to have some security that the code workds

Reply:
We need some assertion to have some security that the code workds

So right now what I'm trying to see if it's creating a jpg with the correct graph, but how can I assert that? it's not just to check that the image was made but that the correct graph was created right?

Reply:
I would try to store some and try to compare picture content when testing